### ISSUE TITLE
expect().to be === does not actually call subject#===

### ISSUE THREAD

**Issue by postmodern** [icy-arctic-fox/spectator#26]
Ran into this when testing custom `#===` methods.

### Example Code

```crystal
module Test
  VERSION = "0.1.0"

  class Divisible

    def initialize(@div : Int32)
    end

    def ===(other : Int32)
      (other % @div) == 0
    end

  end
end
```

```crystal
require "./spec_helper"
require "../src/test"

Spectator.describe Test::Divisible do
  let(i) { 3 }
  subject { described_class.new(i) }

  describe "#===" do
    it "must call === and compare the other object" do
      expect(subject).to be === 6
    end
  end
end
```

### Output

```
$ shards install
Resolving dependencies
Fetching https://gitlab.com/arctic-fox/spectator.git
Using spectator (0.9.36)
$ crystal spec
Test::Divisible
  #===
    must call === and compare the other object

Failures:

  1) Test::Divisible#=== must call === and compare the other object
     Failure: subject does not equal 6

         actual: #<Test::Divisible:0x7f2fbab6abd0 @div=3>
       expected: 6

     # spec/test_spec.cr:9

Finished in 341 microseconds
1 examples, 1 failures, 0 errors, 0 pending
```

### Workaround

However, if we explicitly call `subject#===` within `expect()`, it works:

```crystal
    it "must call === and compare the other object" do
      expect(subject === 6).to be(true)
    end
```

```
$ crystal spec
Test::Divisible
  #===
    must call === and compare the other object

Finished in 104 microseconds
1 examples, 0 failures, 0 errors, 0 pending
```

**Comment by icy-arctic-fox** [comment]
Looks like Spectator is missing some `be` variants that RSpec has. The [docs](https://relishapp.com/rspec/rspec-expectations/v/3-10/docs/built-in-matchers/equality-matchers) don't show `be ===`, but the [code](https://github.com/rspec/rspec-expectations/blob/main/lib/rspec/matchers/built_in/be.rb#L120) does. I'll look into getting this added soon.

**Comment by icy-arctic-fox** [comment]
Sorry for the delay. Should be fixed in the latest version - 0.9.37.

**Comment by postmodern** [comment]
Seems to be working now. Thanks!
